.github/workflows/e2e.yaml

```diff
@@ -144,6 +145,7 @@ jobs:
         with:
           github_token: ${{ secrets.RESTATE_ACTION_READ_TOKEN || secrets.GITHUB_TOKEN }}
           repo: restatedev/restate
+          workflow: 44673365
```

Comment:
Same here. It might be easier to make the connection if we used the workflow file name `docker.yml`.